app/controllers/idv/document_capture_controller.rb

```diff
@@ -19,7 +19,12 @@ def show
       Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
         call('document_capture', :view, true)
 
-      render :show, locals: extra_view_variables
+      case doc_auth_vendor
+      when Idp::Constants::Vendors::SOCURE
+        redirect_to idv_socure_document_capture_url
+      when Idp::Constants::Vendors::LEXIS_NEXIS, Idp::Constants::Vendors::MOCK
+        render :show, locals: extra_view_variables
+      end
```

Comment:
should we add a similar check in the other 3 document capture controllers (hybrid, socure standard, socure hybrid) to redirect them incase they are in the wrong controller ... could be before filters 🤔

Reply:
I don't think that helps much of anything; if they end up in those controllers by accident, we really don't have enough information to figure out where they ought to be. And this is temporary code until we figure out how we're going to fit Socure into our flow on a programatic level; it's unlikely that we'll simply be handing off to one or the other at this point.

Reply:
We know the doc_auth_vendor based on the document_caprltute_session.uuid, right?

Currently as implemented, a user can enter a socure IDV url of and go to it. This should be enforced that a user can't choose their own vendor. 

The interstitial page will be a view for the socire document_capture#show ... maintaining consistency with our other document capture views .

Reply:
I have created an [issue](https://cm-jira.usa.gov/browse/LG-14658) to take this on separately 👍🏿